libs/ett/auth/data-access-security/src/lib/authentication/ett-authentication.interceptor.ts

```diff
@@ -40,7 +40,6 @@ export class EttAuthenticationInterceptor implements HttpInterceptor {
     request: HttpRequest<unknown>,
     nextHandler: HttpHandler
   ): Observable<HttpEvent<unknown>> {
-    request.responseType;
     return nextHandler
       .handle(request)
       .pipe(
```

Comment:
question/suggestion:

Do we know the type of error inside "catchError"?

Reply:
No, but we're asserting the type for the error that this interceptor is handling (`HttpErrorReponse` with HTTP status code `401`). Other errors are delegated to other interceptors and error handlers, hence the `throwError` propagation in line 52.